serverless.yml

```diff
@@ -4,7 +4,7 @@ frameworkVersion: '3'
 provider:
   name: aws
   runtime: nodejs18.x
-  environment: ${file(./env-vars.yml)}
+  environment: ${file(env-vars.js)}
```

Comment:
I couldn't get this to be a TS file without adding a plugin

Reply:
Right, this was mainly the reason why I chose to go with the manual approach [here](https://github.com/api3dao/merkle-funder/issues/7#issue-1770204693) but I wasn't aware that a plugin would allow using a TS script 🤔 

Anyways, if you are OK with using a JS script then this would improve the usability by removing the manual step of generating the env-vars.yml file

Reply:
https://github.com/api3dao/merkle-funder/pull/39/commits/336efc7cb7e3b529808379f9fc3bdb88ce263b7f

Reply:
Yeah, that's my thinking. I think it's a good trade-off.